pkg/apis/v1alpha1/awsnodetemplate.go

```diff
@@ -18,6 +18,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// This will contain the current status of the Subnets and Security Groups of the template
+type AWSNodeTemplateStatus struct {
+	// The Subnet field will contain the current subnet ids that are availbe to the
+	// cluster under the AWSNodeTemplate Subnet selectors. The field will list of strings
+	// with the Subnet id and avaiblity zone
+	// +optional
+	Subnets []string `json:"subnetIds,omitempty"`
```

Comment:
Try to make these fields align with the JSON unless there's a strong reason not to. Same for security groups. WDYT about AMISelector, as well?
```suggestion
	SubnetIDs []string `json:"subnetIds,omitempty"`
```

Reply:
Updated